### ISSUE TITLE
Agreement Amendment CP Extension

### ISSUE THREAD

**Issue by ntrncic** [unicef/etools-issues#915]
Agreement amendment type CP Extension was taken out of the api dropdown. The type is still there for the record but doesn't appear in the frontend. Can we show the agreement type even tho it's not in the dropdown?

**Comment by adi130987** [comment]
Yes, we can show the amendment type for the old records event if it's not on the API anymore and it's saved on agreement. We just have to know the value saved for Amendment Type `CP Extension`. It might be the same text, but I'm not sure.

**Comment by ntrncic** [comment]
the value in the db is `CP extension` in the frontend it should show `Extension of Country Programme Cycle`
